pytorch/inference/docker/2.2/py3/Dockerfile.graviton.cpu

```diff
@@ -0,0 +1,253 @@
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.10.9
```

Comment:
you can upgrade python version to `3.10.13` too tbh. It's been tested with x86 and should work with graviton too since it's just a patch upgrade.